### ISSUE TITLE
--use_existing_masks does not work in run_2_qsm.py script

### ISSUE THREAD

**Issue by loneblues** [QSMxT/QSMxT#86]
Although it appears that there is an argument in when opening the script, it does not seem to work when executing it. 
Is this still a WIP or am i doing something wrong? 

Thanks!

**Comment by astewartau** [comment]
Dear @loneblues, thank you for your question about the `--use_existing_masks` option. The option should be working if your masks are correctly named and stored in the BIDS folder. This is quite a new feature, so bugfixes may have also been made since the version you were using. 

Could you please explain in more detail the problem you had? Was there an error message, or was the mask you created ignored?

You can download an example BIDS folder where the mask file is named correctly using the following:

```bash
# install osfclient for download data from osf
pip install osfclient

# download and extract some test data
mkdir test-data
osf -p 9jc42 clone ./test-data
tar xf ./test-data/osfstorage/bids-osf.tar -C ./test-data

# view the directory tree of the test data
tree test-data
```
```
test-data/
|-- bids-osf
|   |-- README
|   |-- dataset_description.json
|   |-- details_and_citations.txt
|   |-- log_2022-10-20_07-15-05852020.txt
|   `-- sub-1
|       `-- ses-1
|           |-- anat
|           |   |-- sub-1_ses-1_run-01_echo-01_part-mag_MEGRE.json
|           |   |-- sub-1_ses-1_run-01_echo-01_part-mag_MEGRE.nii.gz
|           |   |-- sub-1_ses-1_run-01_echo-01_part-phase_MEGRE.json
|           |   |-- sub-1_ses-1_run-01_echo-01_part-phase_MEGRE.nii.gz
|           |   |-- sub-1_ses-1_run-01_echo-02_part-mag_MEGRE.json
|           |   |-- sub-1_ses-1_run-01_echo-02_part-mag_MEGRE.nii.gz
|           |   |-- sub-1_ses-1_run-01_echo-02_part-phase_MEGRE.json
|           |   |-- sub-1_ses-1_run-01_echo-02_part-phase_MEGRE.nii.gz
|           |   |-- sub-1_ses-1_run-01_echo-03_part-mag_MEGRE.json
|           |   |-- sub-1_ses-1_run-01_echo-03_part-mag_MEGRE.nii.gz
|           |   |-- sub-1_ses-1_run-01_echo-03_part-phase_MEGRE.json
|           |   |-- sub-1_ses-1_run-01_echo-03_part-phase_MEGRE.nii.gz
|           |   |-- sub-1_ses-1_run-01_echo-04_part-mag_MEGRE.json
|           |   |-- sub-1_ses-1_run-01_echo-04_part-mag_MEGRE.nii.gz
|           |   |-- sub-1_ses-1_run-01_echo-04_part-phase_MEGRE.json
|           |   `-- sub-1_ses-1_run-01_echo-04_part-phase_MEGRE.nii.gz
|           `-- extra_data
|               |-- **sub-1_ses-1_run-01_brainmask.nii.gz**
|               |-- sub-1_ses-1_run-01_chi-cropped.nii.gz
|               |-- sub-1_ses-1_run-01_chi-interpolated.nii.gz
|               `-- sub-1_ses-1_run-01_segmentation.nii.gz
`-- osfstorage
    `-- bids-osf.tar
```

With this test data, I tried running the latest version of QSMxT (v1.2.0) with the `--use_existing_masks` option using the following, and the mask seems to have been correctly used. 

```
run_2_qsm.py test-data/bids-osf/ test-data/qsm --use_existing_masks
```

**Comment by loneblues** [comment]
Hi. 

Now that I think of it, it is probably due to the version of QSMxt that I have used, a basic mistake. 
I was using the script based on docker containers, and it appears that there isn't a 1.2 version available currently. 
Sorry for the trouble. 

BTW, the error message that I got regarding the above issue was 

run_2_qsm.py: error: unrecognized arguments: --use_existing_masks

P.S. Is there a plan for v1.2 to be added to dockerhub?
